rocAL/include/meta_data/meta_data.h

```diff
@@ -65,7 +70,8 @@ enum class MetaDataType {
     Label,
     BoundingBox,
     PolygonMask,
-    KeyPoints
+    KeyPoints,
+    AsciiValue
```

Comment:
Why don't we name it Webdataset here

Reply:
Because that structure doesnt store Webdataset.
It stores the values in Ascii format.